python/cuml/test/dask/test_coordinate_descent.py

```diff
@@ -26,9 +26,9 @@
 
 @pytest.mark.mg
 @pytest.mark.parametrize('dtype', [np.float32, np.float64])
-@pytest.mark.parametrize('alpha', [0.1, 0.001])
+@pytest.mark.parametrize('alpha', [0.001])
 @pytest.mark.parametrize('algorithm', ['cyclic', 'random'])
-@pytest.mark.parametrize('nrows', [unit_param(500),
+@pytest.mark.parametrize('nrows', [unit_param(50),
```

Comment:
Change looks good, but I worry a little bit about the small data size could create some brittleness in the test. @Nanthini10 could you use `pytest-repeat` to run the test locally for a number of times (say 100 at least) just to confirm that the test is still robust and for peace of mind?

Reply:
@dantegd  Ran it a 100 times and it passed the tests 👍